websites/M/MarineTraffic/presence.ts

```diff
@@ -0,0 +1,330 @@
+const presence = new Presence({
+		clientId: "1013343014711738368",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let locationMap: string,
+	elemName: string,
+	shipStatus: string,
+	shipDest: string,
+	shipDep: string,
+	elemDesc: string;
+
+const capitalize = (s: string | never[]) =>
+	(s && s[0].toUpperCase() + s.slice(1)) || "";
```

Comment:
Use `charAt()` instead of using indexes
```suggestion
const capitalize = (s: string) =>
	(s && s[0].toUpperCase() + s.slice(1)) || "";
```